packages/catalog-process/src/utilities/fileManager.ts

```diff
@@ -22,12 +40,38 @@ const s3FileManager = (): FileManager => {
     },
     region: config.s3Region,
   });
+
+  const buildS3Key = (
+    path: string,
+    resourceId: string,
+    fileName: string
+  ): string => `${path}/${resourceId}/${fileName}`;
+
   return {
     deleteFile: async (path: string): Promise<void> => {
       await client.send(
         new DeleteObjectCommand({ Bucket: config.s3BucketName, Key: path })
       );
     },
+    copy: async (
+      filePathToCopy: string,
+      documentId: string,
+      fileName: string
+    ): Promise<string> => {
+      logger.info(`Copying file ${filePathToCopy}`);
+
+      const eserviceDocsPath = config.eserviceDocsPath as string;
```

Comment:
we're already using an unsafe cast also at lines 38-39, but I think we should probably track as tech debt to remove those (maybe using a union type for the configuration): I wouldn't do in this PR directly since we already have another branch based on top of this, but for the future I think we shouldn't use unsafe casts if possible

Reply:
tracked as https://buildo.atlassian.net/browse/PPA-27